sidekick/lib/src/init/sidekick_command_runner.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:args/command_runner.dart';
+
+class SidekickCommandRunner extends CommandRunner {
```

Comment:
> Extract sidekick_command_runner.dart so it is more easily found.

Why would you ever search for it?

Reply:
I'd prefer to make it private instead and leave it in the main file

Reply:
> > Extract sidekick_command_runner.dart so it is more easily found.
> 
> Why would you ever search for it?

It was when I still thought, I'd need to manually import it